packages/block-library/src/media-text/edit.js

```diff
@@ -103,9 +104,8 @@ function attributesFromMedia( {
 		if ( mediaType === 'image' ) {
 			// Try the "large" size URL, falling back to the "full" size URL below.
 			src =
-				media.sizes?.large?.url ||
-				// eslint-disable-next-line camelcase
-				media.media_details?.sizes?.large?.source_url;
+				media.sizes?.[ mediaSizeSlug ]?.url ||
+				media?.media_details?.sizes?.[ mediaSizeSlug ]?.source_url;
```

Comment:
This code no longer matches the inline comment above. 

I assume the intent of this change is so that any specified `mediaSizeSlug` is used however now it won't prefer large instead of full as the unchanged comment suggests.